freie-universitat-berlin-geographische-wissenschaften.csl

```diff
@@ -17,35 +17,38 @@
   </info>
   <locale xml:lang="de">
     <terms>
-      <term name="et-al">et al.</term>
-      <term name="editor" form="short"> (Hrsg.)</term>
-      <term name="collection-editor" form="short">(Hrsg.)</term>
-      <term name="page" form="short">S.</term>
+      <term name="et-al">et&#160;al.</term>
       <term name="issue">Heft</term>
     </terms>
   </locale>
   <macro name="edition">
     <choose>
-      <if variable="edition">
-        <group delimiter=" " suffix=" ">
-          <number variable="edition" form="ordinal"/>
-          <label variable="edition" form="short"/>
-        </group>
+      <if is-numeric="edition">
+        <number variable="edition" form="ordinal" suffix=" "/>
+        <label variable="edition" form="short"/>
       </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher-place">
+    <choose>
+      <if variable="publisher-place">
+        <text variable="publisher-place" />
+      </if>
+      <else>
+        <text value=" o.&#160;O."/>
+      </else>
     </choose>
-    <text variable="publisher-place"/>
   </macro>
   <macro name="pagenumbers">
-    <group delimiter=" ">
-      <number variable="number-of-pages"/>
-      <label variable="number-of-pages" form="short"/>
-    </group>
+    <number variable="number-of-pages" suffix=" "/>
+    <label variable="number-of-pages" form="short"/>
```

Comment:
That doesn't look like an improvement. Why did you delete the group? Now it will display the label even the data for the number of pages is not present. There are more such cases following...